Sources/AudioKit/MIDI/MIDINode.swift

```diff
@@ -41,8 +41,8 @@ open class MIDINode: Node, MIDIListener, NamedNode {
     ///   - name: Name to connect with
     ///
     public func enableMIDI(_ midiClient: MIDIClientRef = MIDI.sharedInstance.client,
-                           name: String = "MIDINode") {
-        CheckError(MIDIDestinationCreateWithBlock(midiClient, name as CFString, &midiIn) { packetList, _ in
+                           name: String? = "MIDINode") {
+        CheckError(MIDIDestinationCreateWithBlock(midiClient, name! as CFString, &midiIn) { packetList, _ in
```

Comment:
Making `name` non-optional will remove the need to force-unwrap this, and if it's `nil` (a possibility introduced by making `name` an optional value) this will fail anyway. That is actually a pretty good example of why to avoid using optionals when possible.